### PR TITLE
Feature/1032 error pages

### PR DESCRIPTION
**Pull request by appility**

### CODE REVIEW

File: app/views/errors/not_found.html.haml [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+  .margin-top-30 
+    %p.large If you entered a web address please check it was correct.
+
+    %p.large
+      If you followed a link here
+      = succeed '.' do
+        =link_to 'let us know', "#{Figaro.env.CONTACT_US_FORM}"
+
+    %p.large
+      =link_to 'Go to the homepage', "#{Figaro.env.EX_OPPS_HOMEPAGE}"
```
Comment (agiamas) [inline app/views/errors/not_found.html.haml]: can you change that to Figaro.env.DOMAIN! ? we already store the root url in this variable

File: app/views/errors/not_found.html.haml [hunk 2]
```diff
@@ -1,2 +1,17 @@
 = content_for :errors do
   %h1= t('errors.not_found')
+  .margin-top-30 
+    %p.large If you entered a web address please check it was correct.
+
+    %p.large
+      If you followed a link here
+      = succeed '.' do
+        =link_to 'let us know', "#{Figaro.env.CONTACT_US_FORM}"
```
Comment (agiamas) [inline app/views/errors/not_found.html.haml]: please add this value in all heroku environments, staging and production :)

File: config/locales/en.yml [hunk 3]
```diff
@@ -54,7 +54,7 @@ en:
     subscriptions_section_heading: Your email alerts
     account_section_heading: Your account
   errors:
-    not_found: Sorry, page not found
+    not_found: This page cannot be found
```
Comment (agiamas) [inline config/locales/en.yml]: not a native speaker, but shouldn't it be 
...can not be found...
with a space in there?

File: app/views/errors/not_found.html.haml (hunk 1 above)
Comment (appility) [reply app/views/errors/not_found.html.haml]: Theres no DOMAIN env var on staging

File: config/locales/en.yml (hunk 3 above)
Comment (appility) [reply config/locales/en.yml]: It's "cannot" in the copydex - there's no GDS guidance in their style guide but it does seem to be the modern choice

**Pull request merged**